Release Notes/Release Notes v2.7.1.md

```diff
@@ -17,3 +17,12 @@ http://mongodb.github.io/mongo-csharp-driver/
 Upgrading
 
 There are no known backwards breaking changes in this release.
+
+Important compatibility note:
+
+If your application is running on Linux or OS X and you were planning to upgrade
+to the 2.7.1 release of the driver, you must upgrade to 2.7.2 or later rather than 2.7.1.
+
+See the 2.7.2 release notes for details:
+
+https://github.com/mongodb/mongo-csharp-driver/blob/v2.7.x/Release%20Notes/Release%20Notes%20v2.7.2.md
```

Comment:
Just getting copy of the release notes in master in sync with the actual v2.7.3 release notes.

Reply:
Ack